### ISSUE TITLE
Laptop #1 - wireless dropouts

### ISSUE THREAD

**Issue by Logiqx** [Logiqx/gps-wizard#1]
WiFi has been dropping out and all networks disappearing (dozens of times) and it was also rebooting intermittently.

TLDR – The Qualcomm QCA61x4A wireless NIC seems to cause these dropouts in numerous brands of laptop.

I remember it happening occasionally last year but it caused the 22H2 Windows update to fail last week and the dropouts have been a real pain over the past 2 days. So, I disabled the NIC and plugged in a USB WiFi NIC which has resolved the dropouts, Windows updates have now applied (including 22H2) and the reboots have stopped occurring.

This means the laptop can be left running all day, ready for downloads in the afternoon. 

Some links regarding this network adapter:
- https://www.reddit.com/r/sysadmin/comments/7t57mf/qualcomm_qca61x4a_wireless_issues_anyone_else/
- https://www.catalog.update.microsoft.com/Search.aspx?q=QCA61x4A+windows+10

**Comment by Logiqx** [comment]
Workaround was to disable the WiFi adapter and use a USB dongle.

This is a slightly newer model - https://www.amazon.co.uk/Foktech-802-11ac-Wireless-Network-10-5-10-13/dp/B06XZ5B5G9

Closing issue due to workaround
